### ISSUE TITLE
:skip: no longer works with standard library objects

### ISSUE THREAD

**Issue by eslavich** [astropy/sphinx-automodapi#141]
Prior to the 0.14 release, it was possible to use `:skip:` on standard library objects that were imported by a module:

```
.. automodapi:: sphinx_automodapi.tests.example_module.stdlib
    :skip: time
    :skip: Path
```

In 0.14, those `:skip:` directives are disappeared by code introduced to support `:include:`.  I opened a PR with a test showing the new failure: https://github.com/astropy/sphinx-automodapi/pull/140.  The test passes if I revert the commits from #127.

**Comment by pllim** [comment]
@adrn , any idea how to fix?

**Comment by pllim** [comment]
Although... why does the affected module has no `__all__` defined? See spacetelescope/jwst#6522 for an alternative fix instead of pinning to an older `sphinx-automodapi`.

**Comment by adrn** [comment]
Possible fix in #142
